### PR TITLE
Remove --overwrite in apalacheCurrentPackage task to fix failing builds on MacOS

### PR DESCRIPTION
**Pull request by konnov**
This is a temporary fix for #1368, which lets `make package` go through on my computer. This needs a more thorough analysis though.

It is still producing error messages while unpacking:

```
[info] Unpacking package /Users/.../apalache/target/universal/apalache-0.20.4-SNAPSHOT.tgz to /Users/.../apalache/target/universal
[error] x apalache-0.20.4-SNAPSHOT/
[error] x apalache-0.20.4-SNAPSHOT/LICENSE
[error] x apalache-0.20.4-SNAPSHOT/bin/
[error] x apalache-0.20.4-SNAPSHOT/lib/
[error] x apalache-0.20.4-SNAPSHOT/lib/apalache.jar
[error] x apalache-0.20.4-SNAPSHOT/bin/apalache-mc
[info] Symlinking /Users/.../apalache/target/universal/current-pkg -> /Users/.../apalache/target/universal/apalache-0.20.4-SNAPSHOT
[success] Total time: 8 s, completed Feb 18, 2022 9:42:51 AM

```

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -231,8 +231,8 @@ lazy val root = (project in file("."))
         val (unzipped, _) = IO.split(pkg.toString)
         val target_dir = (Universal / target).value
         val current_pkg = (Universal / target).value / "current-pkg"
-        log.info(s"Unpacking package ${pkg}")
-        s"tar zxvf ${pkg} --overwrite -C ${target_dir}" ! log
+        log.info(s"Unpacking package ${pkg} to ${target_dir}")
+        s"tar zxvf ${pkg} -U -C ${target_dir}" ! log
```
[the remark this answers is not in the document]
Comment (shonfeder) [reply build.sbt:235]: Ah I see now. I think we can just remove this flag, and we don't need the overwrite. tar's default is to overwrite files in any case, these just specify the mechanism. I shouldn't have included it in the first place.

**Comment by shonfeder** [comment]
@konnov do you want to try to build with this branch?

**Review by shonfeder — APPROVED** [review]

**Comment by konnov** [comment]
Thanks! Let me try!

File: build.sbt (hunk 1 above)
[the remark this answers is not in the document]
Comment (konnov) [reply build.sbt]: yeah. I was not sure about `-U` either. Tar has tonnes of flags on different platforms.

**Pull request merged**